vector/build.gradle

```diff
@@ -197,8 +201,8 @@ dependencies {
     /************* flavors management **************/
 
     // app flavor only
-    appCompile 'com.google.firebase:firebase-core:11.6.2'
-    appCompile 'com.google.firebase:firebase-messaging:11.6.2'
+    implementation 'com.google.firebase:firebase-core:11.6.2'
+    implementation 'com.google.firebase:firebase-messaging:11.6.2'
 
```

Comment:
did you notice it is appCompile not compile ? ie it is compiled only for the app flavour.
How do you do it with implementation ?